storybook/src/stories/Global/GlobalUI.stories.tsx

```diff
@@ -16,48 +16,133 @@ export default {
 };
 
 const Container = styled.div`
-  position: fixed;
+  position: absolute;
   top: 0;
   left: 0;
   right: 0;
 `;
 
 const Welcome = () => (
+  <div>
```

Comment:
Did you need a div here specifically just because the output needs a single top level element? 

Not important to update this but a good tip maybe. Usually I use this instead...

```
<>
    /** Child elements in here as usual **/
</>
```

To keep things tidier and I _think_ when it renders this renders everything out at the level the parent would usually be.